build/yamls/antrea-aks.yml

```diff
@@ -691,6 +699,10 @@ rules:
   - patch
   - create
   - delete
+- nonResourceURLs:
+  - /stats/summary
```

Comment:
In commit message, you used "/stat/summary".

Reply:
I just realized using nonResourceURL will not be easy to keep compatible when we change the stats API. 
Switch to k8s style versioned API in latest patch, all API changes are in https://github.com/vmware-tanzu/antrea/pull/1221.

Reply:
Ok.